### PR TITLE
[FIX] l10n_no: set correct default purchase tax

### PR DESCRIPTION
**Pull request by rgo-odoo**
linked to odoo/enterprise#9724

### CODE REVIEW

**Comment by william-andre** [comment]
@jco-odoo Can you check and r+ ?

File: addons/l10n_no/data/account_tax_data.xml [hunk 1]
```diff
@@ -34,6 +34,7 @@
         <record id="tax2" model="account.tax.template">
             <field name="chart_template_id" ref="no_chart_template"/>
             <field name="name">1 Inngående mva høy sats 25%</field>
+            <field name="sequence">0</field>
```
Comment (jco-odoo) [inline addons/l10n_no/data/account_tax_data.xml:37]: @rgo-odoo You can put it as sequence 0, but also put it first in the xml-file please.  Did you check sales side as well?

Comment (rgo-odoo) [reply addons/l10n_no/data/account_tax_data.xml:37]: @jco-odoo sales has default 25% already.

**Comment by jco-odoo** [comment]
@robodoo r+

**Comment by jco-odoo** [comment]
@rgo-odoo Should not we do it in v13 as well?  (and saas-13.2 still exists?)

**Comment by rgo-odoo** [comment]
@jco-odoo nope, V13 already has correct default taxes. Not sure about saas-13.2 but this issue originated from <a href="https://github.com/odoo/odoo/commit/ff9efe7039ac5b4d3d89671828e535833c386392">saas-13.2</a> only so I proposed in this branch.

**Pull request closed without merging**